src/utils/daos.c

```diff
@@ -560,9 +578,10 @@ common_op_parse_hdlr(int argc, char *argv[], struct cmd_args_s *ap)
 	char			*cmdname = NULL;
 
 	assert(ap != NULL);
-	ap->p_op = -1;
-	ap->c_op = -1;
-	ap->o_op = -1;
+	ap->p_op  = -1;
```

Comment:
i don't know why you added a space for the first free. maybe it's a tab and github isn't showing this properly.